.env.sample

```diff
@@ -22,7 +23,7 @@ DB_PORT=3306
 DB_HOST=127.0.0.1
 DB_DATABASE=standard_notes_db
 DB_USERNAME=std_notes_user
-# Please change this!
+# Please change this!!!
```

Comment:
Lol what's this supposed to accomplish? Please revert this part so we can review the other line independently.

Reply:
Done :)